.github/workflows/docker.yml

```diff
@@ -5,45 +5,133 @@ on:
 
 env:
   CI_RUST_TOOLCHAIN: 1.70.0
+  REGISTRY_IMAGE: datenlord/xline
 
 jobs:
-  docker:
-    name: Docker
-    runs-on: ubuntu-latest
+  push_image:
+    name: Push Docker Image
+    runs-on: ${{ matrix.job.os }}
```

Comment:
While we can run jobs on vm that gihub provides, why do we need the `cross` crate for compilation?

Reply:
Currently, [GitHub-hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) are all x86_64 architecture. If we need to support ARM, we need to cross compile, or use ARM architecture Self-hosted runners.